src/nova.app.src

```diff
@@ -7,7 +7,7 @@
   {applications,[
                  kernel,
                  stdlib,
-                 sasl,
+                 %%sasl,
```

Comment:
Why not remove this?

Reply:
Should not be removed, was unintentionally adding this to stage :)